packages/quantic/force-app/main/default/lwc/quanticDocumentSuggestion/quanticDocumentSuggestion.js

```diff
@@ -39,14 +41,16 @@ export default class QuanticDocumentSuggestion extends LightningElement {
    * Whether or not we want to disply the quick view for the document suggestions.
    * @api
    * @type {boolean}
+   * @defaultValue `false`
    */
-  @api showQuickview = false;
+  @api hideQuickview = false;
   /**
-   * Whether or not we want to fetch suggestions when initializing this component.
+   * Whether or not we want to prevent fetching suggestions when initializing this component.
    * @api
    * @type {boolean}
+   * @defaultValue `false`
    */
-  @api fetchOnInit = false;
+  @api preventFetchOnInit = false;
```

Comment:
Removing `fetchOnInit` is a breaking change. I suggest we manage to honor the now deprecated property.

Reply:
What are the possible options we can do to handle such cases when we need to make breaking change?

Reply:
Unfortunately, I'm not yet sure what is the best way to handle breaking changes with LWC and unmanaged packages.

Reply:
Okay, so I think I'll just revert the changes about #fetchOnInit and #showQuickview just so I dont block this PR needed by the cookbook, and I will come back and update these two properties once we figure out how we will handle the breaking changes in quantic, what do u think?

Reply:
I agree, reverting those breaking changes seems to be the way to go for now.